candig/server/datamodel/variants.py

```diff
@@ -601,9 +601,11 @@ def _updateVariantAnnotationSets(self, variantFile, dataUrl):
                         # TODO we need _much_ more sophisticated processing
                         # of VEP versions here. When do they become
                         # incompatible?
-                        if version == "v82":
+                        if "v82" in version:
```

Comment:
if there a particular reason for using the `in` operator here? i can see a use case where if you say `if version in ["v82", "v98"]` this might possibly save a couple lines here..

Reply:
So apparently when I tried to run my annotations through, the string was actually in quotes, like `"v98"`, so I had a really hard time realizing why `'"v98"' == "v98"` kept returning False! I switched from the equality operator to the in operator because it captures substring matches.

The reason there is a separate elif for v98 is because that one is a hack.

Reply:
Makes sense!